documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -84,6 +73,15 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
     cd RMG-Py
     conda env create -f environment.yml
 
+.. warning:: Apple Silicon (M1+) Users
```

Comment:
Is this the same suggestion for Intel Mac?

Reply:
No, it only applies to M1 users (cc @calvinp0 who figured this out, thanks!)